docs/adrs/0004-bulk-import-output-cells.md

```diff
@@ -0,0 +1,48 @@
+# Bulk Import Output Cells
+
+Date: 2020-10-12
```

Comment:
You wrote this last year?

Reply:
It was well on my mind. I wrote it in my head. Definitely.

Reply:
Fixed. :P

Reply:
Well, I plan on revising it until next Tuesday. So that's fitting.
But I'll revert it to when I made the PR, I GUESS.